src/org/mozilla/javascript/Context.java

```diff
@@ -349,6 +349,26 @@
      */
     public static final int FEATURE_ENABLE_XML_SECURE_PARSING = 20;
 
+    /**
+     * Configure whether the entries in a Java Map can be accessed by properties.
+     *
+     * WARNING: This feature is similar to the one in Nashorn, but incomplete.
+     *
+     * 1. A entry has priority over method.
+     *
+     *   map.put("put", "abc");
+     *   map.put;  // abc
+     *   map.put("put", "efg"); // ERROR
+     *
+     * 2. The distinction between numeric keys and string keys is ambiguous.
+     *
+     *   map.put('1', 123);
+     *   map['1']; // Not found. This means `map[1]`.
+     *
+     * @since 1.7 Release 14
+     */
```

Comment:
Can you add to this description a bit so that we can show what the flag does with a small example? Or we can write it up in the release notes or steal from the previous feature. I'd imagine we'd show an example like:

Before:
  var m = new java.util.HashMap();
  m.put('foo', 1);
  print(m.foo)
    undefined

After:
  var m = new java.util.HashMap();
  m.put('foo', 1);
  print(m.foo)
    1

Reply:
4322742 How about this?

Reply:
Great. Thanks!